File: archives/python/2476.py
N = int(input())
arr = []
for _ in range(N):
    a, b, c = map(int, input().split())
    if a == b and b == c and a == c: arr.append(10000+a*1000)
    elif (a == b or a == c) : arr.append(1000+a*100)
    elif (b == c): arr.append(1000+b*100)
    else: arr.append(max(a, b, c)*100)
print(max(arr))

File: archives/cpp/2476.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    ios_base::sync_with_stdio(false);
    cin.tie(NULL);
    cout.tie(NULL);
    
    int max_num = 0;
    int N;
    cin >> N;
    int arr[3];
    while (N--) {
        for (int i = 0; i < 3; i++){
            cin >> arr[i];
        }

        if (arr[0] == arr[1] && arr[1] == arr[2]) {
            max_num = max(max_num, 10000 + arr[0] * 1000);
        } else if(arr[0] == arr[1]) {
            max_num = max(max_num, 1000 + arr[1] * 100);
        } else if (arr[1] == arr[2]) {
            max_num = max(max_num, 1000 + arr[1] * 100);
        } else if (arr[0] == arr[2]) {
            max_num = max(max_num, 1000 + arr[0] * 100);
        } else {
            max_num = max(max_num, *max_element(arr, arr + 3) * 100);
        }
    }
    
    cout << max_num;
    
    return 0;
}